templates/web/shropshire/report/new/roads_message.html

```diff
@@ -0,0 +1,10 @@
+<div id="js-roads-responsibility" class="box-warning hidden">
+    <strong>Not maintained by Shropshire Council</strong>
+    <div id="js-not-a-road" class="hidden js-responsibility-message">
+        <p>The location you have selected doesn't appear to be on <span class="js-roads-asset" data-original="a road">a public road</span> that Shropshire Council maintain.</p>
+        <p>Please select <span class="js-roads-asset" data-original="a road">a road</span> on which to make a report.</p>
+    </div>
+    <div id="js-roads-private" class="js-responsibility-message hidden">
```

Comment:
For this to work on .com, it needs including in the fixmystreet.com equivalent template also. Or you could use `js-not-council-road` instead of `js-roads-private` and then it would use that existing div there, which may be similar enough.